client/configuration/lint-staged/lint-staged.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  "./src/**/*.js": "eslint --fix ./src/",
```

Comment:
Thanks for adding this, @seve 🎉 👏 !!

Do we wanna run `prettier` on staged files too?

And in Meta we added `git add` to add the fixed files into the commit

e.g., 

```ts
'*.js': ['prettier --write', 'git add'],
```

Do we want to do it here too?

Thank you!

https://github.com/chanzuckerberg/khepri/blob/develop/lint-staged.config.js

Reply:
We currently don't have prettier added as a dev dep, however since we are using the prettier ESLint plugin, `--fix` will auto-format for us similarly to how prettier --write' would. The question is: do we have to git add after running `eslint --fix`.
> The rule is autofixable -- if you run eslint with the --fix flag, your code will be formatted according to prettier style. [(link)](https://github.com/prettier/eslint-plugin-prettier)

Thank's for the feedback!  @tihuan

Reply:
[It seems like it automatically adds it to the commit.](https://github.com/okonet/lint-staged#automatically-fix-code-style-with---fix-and-add-to-commit)

Reply:
Ah nice!! Thanks for the link 😄  Glad to know `git add` is no longer needed. Will update on the Meta too!